### ISSUE TITLE
Training objective of moirai-moe

### ISSUE THREAD

**Issue by huyz1117** [SalesforceAIResearch/uni2ts#205]
When training the moirai-moe model, the loss function has a negative log-likelihood. Is there any auxiliary loss function used (to solve the gating function load balancing issue)?

**Comment by liuxu77** [comment]
Hi @huyz1117, we did not use auxiliary loss, although we explored using load balancing loss as in the ablation study in the paper.
